index.js

```diff
@@ -45,13 +45,20 @@ const applyErrorMiddlewares = (app, config, i18n) => {
   }));
 };
 
-const getContentSecurityPolicy = (csp) => {
+const getContentSecurityPolicy = (config) => {
+  let csp = config.csp;
+  if (config.gaTagId) {
+    csp = {
+      scriptSrc: ['http://www.google-analytics.com/analytics.js'],
+      imgSrc: ['http://www.google-analytics.com/collect']
+    };
+  }
```

Comment:
I don't think this is correct. It's blowing away any user-provided config for csp if `gaTagId` is set.

In particular, if a user has provided custom csp *and* a `gaTagId` then both should be respected, and the google analytics sources should be added to the user-provided csp.